### ISSUE TITLE
Runtime error

### ISSUE THREAD

**Issue by SanjidHossain** [gradio-app/gradio#6484]
### Describe the bug

Few of my(4) api created in gradio spaces are showing runtime error, While another api is working fine, I have checked and double checked my .py . one of those 4 api script is as same as the one that is working. So I am not sure why I am facing this issue.since if I had issue with my code then none of my model should have worked

up untill a few days ago all of my api were working fine , I just noticed the issue today

### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Reproduction

import gradio as gr
import onnxruntime as rt
from transformers import AutoTokenizer
import torch, json

tokenizer = AutoTokenizer.from_pretrained("distilroberta-base")

with open("label_types_encoded.json", "r") as fp:
    encode_genre_types = json.load(fp)

genres = list(encode_genre_types.keys())

inf_session = rt.InferenceSession('food-origin-classifier-quantized.onnx')
input_name = inf_session.get_inputs()[0].name
output_name = inf_session.get_outputs()[0].name

def classify_food_Ingredient(article):
    input_ids = tokenizer(article)['input_ids'][:512]
    logits = inf_session.run([output_name], {input_name: [input_ids]})[0]
    logits = torch.FloatTensor(logits)
    probs = torch.sigmoid(logits)[0]
    return dict(zip(genres, map(float, probs)))

label = gr.outputs.Label(num_top_classes=5)
iface = gr.Interface(fn=classify_food_Ingredient, inputs="text", outputs=label)
iface.launch(inline=False)

### System Info

```shell
My internal enviroment is not the issue, it is already been deplyoed on hugging face spaces using gradio and was working fine
```

### Severity

I can work around it

**Comment by abidlabs** [comment]
Hi @SanjidHossain can you link to the Space? What error are you seeing? What output do you expect to see? 

This issue does not have enough details for us to be able to help.

**Comment by abidlabs** [comment]
Closing as we don't have details to reproduce this, but we can reopen once we do.

**Comment by SanjidHossain** [comment]
Thank you for the response, I have solved the problem, Requirments.txt
file  stopped working in gradio spaces, I had to manually resave the txt
file with just small change.
